jcl/src/openj9.traceformat/share/classes/com/ibm/jvm/traceformat/TraceFormat.java

```diff
@@ -383,7 +383,7 @@ public static void main(String[] args) throws Exception {
 
 		context.message(context, "Completed processing of " + context.getTotalTracePoints() + " tracepoints with " + context.getWarningCount() + " warnings and " + context.getErrorCount() + " errors");
 		if (verbose.booleanValue()) {
-			end = System.currentTimeMillis();
+			end = System.nanoTime();
 			float Mbps = (float) (recordsInData * context.getRecordSize()) / (float) (((float) (end - start) / 1000) * (1024 * 1024));
 			context.message(context, "Total processing time " + (end - start) + "ms (" + Mbps + "Mb/s)");
```

Comment:
The message indicates "ms", but now it's "ns".

Reply:
Updated.